index.html

```diff
@@ -4,34 +4,80 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Frontawesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+      integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio mobile version</title>
   </head>
   <body>
     <span class="cercle1"></span>
     <span class="cercle2"></span>
-
-    <section>
+    <header>
+      <h1><a href="#">JK</a></h1>
       <nav>
-        <h1>JK</h1>
         <div class="toggle">
-          <span></span>
+          <a href="#"><span></span></a>
         </div>
```

Comment:
[REQUIRED] Although I do appreciate the way you handled the `hamburger button` here :+1: :sunglasses: I have to ask you to follow the instructions given by the previous code reviewer which states using mob-menu (hamburger) icon instead of using `span` and doing a trick with `: before `and `: after` selectors. 

[OPTIONAL] You can improve it even more, since it is a button (its main job is to invoke an event or function etc), in this particular case it is to toggle the `mob-nav bar`. So why don't we try to use already build in tag `<button />` in this case instead of anchor tag `<a />` (note: anchor tags are mainly used for internal and external links)? :relaxed: . Sample code:

```HTML
<button type="button">
  <img src="icon_path" alt="mob menu icon" />
</button>
```